examples/User Creation & Provisioning/README.md

```diff
@@ -33,19 +33,57 @@ This PowerShell script uses the Box CLI to build and create an onboarding folder
 
 4. Create an OAuth Application using the [CLI Setup Quick Start][oauth-guide].
 
-## 1. Script Parameters
-1. Update the [EmployeeList](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L10) to set your own Employee List CSV Path.
-2. Customize the input file of employee accounts for creation.
-    For example, update the [employees_5.csv](/examples/User%20Creation%20%26%20Provisioning/Employees_5.csv) with the following data:
-    ```
-    firstName,lastName,email
-    Managed,User 1,ManagedUser1@test.com
-    Managed,User 2,ManagedUser2@test.com
-    Managed,User 3,ManagedUser3@test.com
-    ```
-3. Update the [RootFolderParentID](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L21) to set destination folder ID for your changes, either when using JSON file as input to create folder structure, or uploading local structure. It is set to `0` by default, but feel free to set it to your needs.
-4. Update the [FolderStructureJSONPath](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L13) to set your own Folder Structure JSON path. You can also change the [RootFolderName](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L17). It's a name of the folder, that will be created as parent for folders from JSON structure. It's set to `Onboarding` by default, but feel free to set it to your needs.
-5. Optional: To directly upload folder structure from local directory, update [LocalUploadPath](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L14), comment out [New-Folder-Structure](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L113), and uncomment [RootFolderID](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L116-L117). 
+## 1. Update the parameters
```

Comment:
There is a repetition in Headers: 

- ## 1. Update the parameters
- ### Updating parameters

Suggest the following:

- ## Configure the script
- ### Define script parameters